src/main/java/cuchaz/enigma/translation/mapping/serde/MappingFormat.java

```diff
@@ -14,7 +14,7 @@
 public enum MappingFormat {
 	ENIGMA_FILE(EnigmaMappingsWriter.FILE, EnigmaMappingsReader.FILE),
 	ENIGMA_DIRECTORY(EnigmaMappingsWriter.DIRECTORY, EnigmaMappingsReader.DIRECTORY),
-	TINY_V2(null, new TinyV2Reader()),
+	TINY_V2(new TinyV2Writer("intermediary", "named"), new TinyV2Reader()),
```

Comment:
Maybe the columns shouldn't be hardcoded. Someone may use Enigma to remap non-Minecraft stuff and they may want different column names. Don't we have a "mapping 
 options" class that's passed to the mapping readers and writers? The column names could go there instead.

Reply:
In the command line interface they can specify like `tinyv2:input:output` format to specify input and output namespaces. This is only for the in-gui writer.